src/modules/dashboard/index.tsx

```diff
@@ -94,121 +110,180 @@ const Dashboard = () => {
     return (
       categoriesList?.map((cate) => ({
         label: cate.name,
-        value: cate.name,
+        value: cate.id,
       })) || []
     );
   }, [categoriesList]);
   const { data: departmentsList, isLoading: departmentsFetching } =
     useDepartments(1);
-
+  console.log(departmentsList, 'list');
   const deptOptions = useMemo(() => {
     return (
       departmentsList?.map((dept) => ({
         label: dept.name,
-        value: dept.name,
+        value: dept.id,
       })) || []
     );
-  }, [departmentId, departmentsList]);
+  }, [departmentsList]);
 
   const updatedData = data?.slice(page * rowsPerPage, (page + 1) * rowsPerPage);
 
   const updateddataSearch = useMemo(() => {
```

Comment:
Follow proper camelcasing here. should be updatedDataSearch or updatedSearchData sounds more relevant